snappiershot/serializers/constants.py

```diff
@@ -178,12 +180,31 @@ class CustomEncodedPathTypes(_CustomEncodedTypeCollection):
     )
 
 
+class CustomEncodedUnitTypes(_CustomEncodedTypeCollection):
+    """Collection of custom-encoded Unit types from the pint package."""
+
+    # Corresponds to the _CustomEncodedType.type_key attribute.
+    type_key = "__snappiershot_unit__"
+    # Corresponds to the _CustomEncodedType.value_key attribute.
+    value_key = "value"
+
+    unit = _CustomEncodedType(
+        type_=Unit, name="Unit", type_key=type_key, value_key=value_key
+    )
+
+
 # Tuples of types intended to be used for isinstance checking.
 PRIMITIVE_TYPES = bool, float, int, type(None), str
 COLLECTION_TYPES = tuple(value.type for value in CustomEncodedCollectionTypes.list())
 DATETIME_TYPES = tuple(value.type for value in CustomEncodedDatetimeTypes.list())
 NUMERIC_TYPES = tuple(value.type for value in CustomEncodedNumericTypes.list())
 PATH_TYPES = PurePosixPath, PureWindowsPath, Path
+UNIT_TYPES = tuple(value.type for value in CustomEncodedUnitTypes.list())
```

Comment:
i like this so much better than extraneous line breaks in other_TYPES definitions above. If you're feeling generous, could you remove those above?